tests/app/v2/notifications/test_notification_schemas.py

```diff
@@ -0,0 +1,91 @@
+import uuid
+
+import pytest
+from flask import json
+from jsonschema import ValidationError
+
+from app.v2.notifications.notification_schemas import post_sms_request, post_sms_response
+from app.schema_validation import validate
+
+valid_json = {"phone_number": "07515111111",
+              "template_id": str(uuid.uuid4())
+              }
+valid_json_with_optionals = {
+    "phone_number": "07515111111",
+    "template_id": str(uuid.uuid4()),
+    "reference": "reference from caller",
+    "personalisation": {"key": "value"}
+}
+
+
+@pytest.mark.parametrize("input", [valid_json, valid_json_with_optionals])
+def test_post_sms_schema_is_valid(input):
+    validate(input, post_sms_request)
+
+
+def test_post_sms_json_schema_bad_uuid_and_missing_phone_number():
+    j = {"template_id": "notUUID"}
+    try:
+        validate(j, post_sms_request)
+    except ValidationError as e:
```

Comment:
better to use `with pytest.raises(ValidationError) as e:` else this test will silently pass if it doesn't err